servers/services/src/main/java/org/projectnessie/services/cel/CELTranslator.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.services.cel;
+
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import java.util.List;
+import java.util.Set;
+import java.util.StringJoiner;
+import java.util.stream.Collectors;
+import org.projectnessie.api.params.CommitLogParams;
+import org.projectnessie.api.params.EntriesParams;
+import org.projectnessie.model.Contents.Type;
+
+/**
+ * The purpose of the {@link CELTranslator} is to translate given filtering parameters, such as
+ * {@link CommitLogParams} / {@link EntriesParams} into valid CEL expressions as defined in the <a
+ * href="https://github.com/google/cel-spec/blob/master/doc/langdef.md">CEL language definition</a>.
+ *
+ * @see <a href="https://github.com/google/cel-spec/blob/master/doc/langdef.md">CEL language
+ *     definition</a>
+ */
+public class CELTranslator {
+
+  public static String from(CommitLogParams params) {
```

Comment:
I wonder if this is cleaner as a `Visitor` pattern or something? This will get big and hairy w/ parameter additions.

Reply:
so far I tried it to keep it as simple as possible for the prototype. I think once we support more and more params that we translate to CEL, we'll eventually rewrite this and make it more flexible

Reply:
makes sense to me :+1: